.github/workflows/promptflow-global-config-test.yml

```diff
@@ -0,0 +1,113 @@
+name: promptflow-global-config-test
+on:
+  schedule:
+    - cron: "40 18 * * *" # Every day starting at 2:40 BJT
+  pull_request_target:
```

Comment:
When test, use pull_request trigger to test your own pr.
When check in, please change back to pull_request_target.